runelite-client/src/main/java/net/runelite/client/ui/overlay/OverlayStyle.java

```diff
@@ -22,17 +22,17 @@
  * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-package net.runelite.client.plugins.npchighlight;
+package net.runelite.client.ui.overlay;
 
-public enum RenderStyle
+public enum OverlayStyle
```

Comment:
I would name this highlightstyle, overlaystyle seems too confusing. Like it has anything to do with overlay system, but it dont